pkg/osa/create.go

```diff
@@ -4,18 +4,21 @@ import (
 	"io/ioutil"
 	"os"
 
-	"github.com/jim-minter/azure-helm/pkg/arm"
-	"github.com/jim-minter/azure-helm/pkg/config"
-	"github.com/jim-minter/azure-helm/pkg/helm"
 )
 
-func Create() error {
-	m, err := readManifest()
+type RP struct {}
+
+func (*RP) Run() error {
```

Comment:
this is what I'd consider our "simulator".  Same should hold true for upgrade.go